cmdb/account/{{cookiecutter.account_name}}/config/settings/shared/settings.json

```
@@ -1,5 +1,39 @@
 {
     "Registries" : {
-
+        "docker" : {
+            "EndPoint" : "{{cookiecutter.aws_account_id}}.dkr.ecr.ap-southeast-2.amazonaws.com"
+        },
```

This registry remains provider specific. Could we wrap in logic based on the Provider?

```suggestion
        {% if cookiecutter.account_type == "aws" %}
        "docker" : {
            "EndPoint" : "{{cookiecutter.aws_account_id}}.dkr.ecr.ap-southeast-2.amazonaws.com"
        },
        {% endif %}
```